ci/env/kava-protonet/KAVA.VERSION

```diff
@@ -1 +1 @@
-47416b3d1af6dfd31bab479d78b24cdbb677be31
+master
```

Comment:
@evgeniy-scherbina I saw you pinned this commit recently, is it ok to change back to master?

Reply:
yes